docs/resources/Guild.md

```diff
@@ -370,7 +393,7 @@ Remove a member from a guild. Requires 'KICK_MEMBERS' permission. Returns a 204
 
 ## Get Guild Bans % GET /guilds/{guild.id#DOCS_GUILD/guild-object}/bans
 
-Returns a list of [user](#DOCS_USER/user-object) objects that are banned from this guild. Requires the 'BAN_MEMBERS' permission.
+Returns a list of [ban](#DOCS_GUILD/ban-object) objects for the users banned from this guild. Requires the 'BAN_MEMBERS' permission.
```

Comment:
This will be a broken link. You can rename `### Guild Ban Object` to `### Ban Object` and this will be fine. Should also rename `Guild Ban Structure` to `Ban Structure` as well.